src/main/kotlin/frc/robot/subsystems/shooter/conveyor/Conveyor.kt

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.shooter.conveyor
+
+import com.ctre.phoenix6.controls.VoltageOut
+import edu.wpi.first.units.measure.Voltage
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+import frc.robot.lib.extensions.volts
+
+import frc.robot.lib.logged_output.LoggedOutputManager.runOnce
+import frc.robot.lib.universal_motor.UniversalTalonFX
+import org.littletonrobotics.junction.Logger
+import org.team5987.annotation.LoggedOutput
+
+object Conveyor : SubsystemBase() {
+ private val motor =
+     UniversalTalonFX(port = PORT , gearRatio = GEAR_RATIO, config = CONFIG , simGains = SIM_GAINS  )
+    @LoggedOutput  var voltageSetpoint = 0.volts
+    private var voltageRequest : VoltageOut = VoltageOut(0.0)
+
+    fun setVoltage (voltage : Voltage) = run{
```

Comment:
Sorry for the confusion. This indeed should be a `runOnce`.

```suggestion
    fun setVoltage (voltage : Voltage) = runOnce {
```